docs/source/tutorial.rst

```diff
@@ -57,6 +62,12 @@ Response will contain ``access.token`` for the contract that can be used for fur
 .. include:: tutorial/contract-credentials.http
    :code:
 
+2. Second method
+~~~~~~~~~~~~~~~~
+For generating new contract credentials is used Transfer object.
+
+* `Contracts credentials change with Transfer object (Relocation) <http://relocation.api-docs.openprocurement.org/en/latest/tutorial.html#contract-credentials-change>`_
```

Comment:
For more detailed instructions see `Contracts credentials change with Transfer object <http://relocation.api-docs.openprocurement.org/en/latest/tutorial.html#contract-credentials-change>`_ in the Relocation documentation.